docs/kb/integrations/defect-dojo-integration.md

```diff
@@ -126,6 +126,17 @@ In the example, the values are:
 * `PRODUCT` = chess-game
 * `IMPORT_SEMGREP_TO_DEFECTDOJO_SHA_CHECKSUM` = c41aed4055adeee415b795cc17a069b144fb51bc31f6c4925be3b82d0b54de33 Uimport_semgrep_to_defect_dojo.py
 
+The content for this last variable was generated with the following command:
+`shasum -a 256 -U import_semgrep_to_defect_dojo.py`
+it will generate an unique key taking as input the content of the script and it will be used to verify the integrity. 
+
+In the pipeline, the integrity of the script is verified with the following commands:
+```
+echo $IMPORT_SEMGREP_TO_DEFECTDOJO_SHA_CHECKSUM > sha-import-dd.tmp
+shasum -a 256 -U -c sha-import-dd.tmp
+```
+If the script has not been tampered, the pipeline will continue normal execution, otherwise it will stop and return an error.
```

Comment:
```suggestion
If the script has not changed since the checksum was generated, the pipeline will continue normal execution, otherwise it will stop and return an error.
```